rclcpp/test/rclcpp/test_time.cpp

```diff
@@ -451,6 +452,18 @@ TEST_F(TestTime, test_overflow_underflow_throws) {
     std::underflow_error("addition leads to int64_t underflow"));
 }
 
+TEST_F(TestTime, validity) {
+  EXPECT_FALSE(rclcpp::Time(0, 0, RCL_CLOCK_UNINITIALIZED).is_valid());
+  EXPECT_FALSE(rclcpp::Time(0, 0, RCL_SYSTEM_TIME).is_valid());
+  EXPECT_FALSE(rclcpp::Time(0, 0, RCL_STEADY_TIME).is_valid());
+  EXPECT_FALSE(rclcpp::Time(0, 0, RCL_ROS_TIME).is_valid());
+
+  EXPECT_TRUE(rclcpp::Time(0, 1, RCL_CLOCK_UNINITIALIZED).is_valid());
```

Comment:
i am inclined to make this FALSE, since it does not mean valid time because it is not initialized? what this time means valid to us? @ros2/team the same comment goes to https://github.com/ros2/rcl/pull/1018#pullrequestreview-1172904338. what does it mean valid for this time w/o initialization? for me, it is invalid...